tools/e2etesting/DeployEdge.ps1

```diff
@@ -93,6 +102,18 @@ Set-AzKeyVaultSecret -VaultName $keyVault.VaultName -Name 'iot-edge-device-id' -
 Write-Host "Updating 'os' and '__type__'-Tags in Device Twin..."
 Update-AzIotHubDeviceTwin -ResourceGroupName $ResourceGroupName -IotHubName $iotHub.Name -DeviceId $edgeIdentity.Id -Tag @{ "os" = "Linux"; "__type__" = "iiotedge"} | Out-Null
 
+## Generate SSH keys
+$privateKeyFilePath = [System.IO.Path]::Combine($KeysPath, "id_rsa_iotedge")
```

Comment:
Join-Path exists in PowerShell as well. :-)

Reply:
updated